src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IUnaryOperatorExpression.cs

```diff
@@ -2899,5 +2899,165 @@ unsafe void Method()
 
             VerifyOperationTreeForTest<PrefixUnaryExpressionSyntax>(source, expectedOperationTree);
         }
+
```

Comment:
Can you add some tests for the scenario of `nullable + non-nullable`? Also some tests involving `nullable + null`.